.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: umidbekk/actions/prepare-node-repo@v2
         with:
           cache-key: yarn-v4
-
+          node-version: 16
```

Comment:
@tokarevfedor1991 Were there any problems without this line?

Reply:
yes, Github uses node 18 for build, so our project is not working

Reply:
I think we should also support node 18 in our projects
I will add this point to our meeting notes